File: backend/src/main/java/com/tododuk/domain/team/dto/TeamMemberAddRequestDto.java
package com.tododuk.domain.team.dto;

import com.tododuk.domain.team.constant.TeamRoleType;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class TeamMemberAddRequestDto {
    @NotNull(message = "이메일은 필수입니다.")
    private String email; // 이메일로 사용자 찾기

    @NotNull(message = "역할은 필수입니다.")
    private TeamRoleType role; // ex) leader, member
}

File: backend/src/main/java/com/tododuk/domain/team/dto/TeamMemberAddRequestDto.kt
package com.tododuk.domain.team.dto

import com.tododuk.domain.team.constant.TeamRoleType
import jakarta.validation.constraints.NotNull

data class TeamMemberAddRequestDto(
    @field:NotNull(message = "이메일은 필수입니다.")
    val email: String, // 이메일로 사용자 찾기

    @field:NotNull(message = "역할은 필수입니다.")
    val role: TeamRoleType // ex) leader, member
)